tests/integration/auth_adapter/test_sessions.py

```diff
@@ -437,22 +437,34 @@ async def test_login_with_cookie_and_registered_data_steward(bare_client: BareCl
     )
 
 
-async def test_login_with_cookie_and_invalid_csrf_token(bare_client: BareClient):
-    """Test login request with session cookie and invalid CSRF token."""
+async def test_login_with_cookie_but_without_csrf_token(bare_client: BareClient):
+    """Test login request with session cookie but no CSRF token.
+
+    It should be possible to login from a new browser tab using the session cookie.
+    """
+    setup_daos()
+
     store = await get_session_store(config=CONFIG)
-    session = await store.create_session(
-        ext_id="john@aai.org", user_name="John Doe", user_email="john@home.org"
+    session_dict = await store.create_session(
```

Comment:
Not sure why this is called `session_dict` now, got confused for a second with the attribute access below

Reply:
I think I copied it from another test that had this misleading name. Fixed in 8bc136a.